Development

ramucv0527 opened the pull request:
Before issuing a pull request, please see the contributing page.

rhsftc committed:
Installed FtcLib and added omni/mecanum test op mode.

Jok1o7 committed:
Change

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Motor name change

Jok1o7 committed:
Drive code

This is the commited drive code for our base teleop

Jok1o7 committed:
Deleted one useless file

Jok1o7 committed:
Refactoring the Robot drive and hardware code

AlexanderBecker13 committed:
scope of a variable

AlexanderBecker13 committed:
Programed the First Autonomous Parts

We got the robot to go forward without using a controller. We didn't get it to stop yet.

AlexanderBecker13 committed:
Update SDK to 9.0.1

AlexanderBecker13 committed:
Gradle Update

rhsftc committed:
Refactored MotorControl and added FtcLib GamepadEx to CenterstageDrive teleop.

rhsftc committed:
Refactored MotorControl and added FtcLib GamepadEx to CenterstageDrive teleop.

rhsftc committed:
Converted DcMotors to DcMotorEx.

rhsftc committed:
Changed motors to FLOAT mode.

rhsftc committed:
Added ImuControl class to mangage access to the IMU.

rhsftc committed:
One more Gradle update.

Jok1o7 committed:
Update CenterstageDrive.java

Jok1o7 committed:
Roadrunner tuning add

Jok1o7 committed:
Added implementations in gradle

Jok1o7 committed:
roadrunner works

AlexanderBecker13 committed:
Added MeepMeep

AlexanderBecker13 committed:
changed MeepMeep field to Centerstage

Jok1o7 committed:
Refactored project folders

Jok1o7 committed:
Merge branch 'Roadrunner' of https://github.com/edinagoinglikehotcakes/centerstage into Roadrunner

rhsftc committed:
AGP update to 8.1.2

rhsftc committed:
Merge remote-tracking branch 'origin/Roadrunner' into rhs

rhsftc committed:
Playing with RR in MeepMeep

AlexanderBecker13 committed:
SDK to 9.0.1

AlexanderBecker13 committed:
Gradle Update

AlexanderBecker13 committed:
More Gradle Fixes

rhsftc committed:
Revert "More Gradle Fixes"

This reverts commit 35ccc9deb95e427107bda95efb460515aceb2690.

rhsftc committed:
Revert "Gradle Update"

This reverts commit 24464782a5db64b0e23f4d9022a1d9cbe6a9bd62.

rhsftc committed:
Revert "SDK to 9.0.1"

This reverts commit 7e43ce5bfef74ff74553c9d062d7272ea6cd550f.

rhsftc committed:
Updated SDK to 9.0.1

rhsftc committed:
Merge remote-tracking branch 'origin/Roadrunner' into rhs

rhsftc committed:
Change to Reflect new Hotcakes package

AlexanderBecker13 committed:
Updated the inPerTick and lateralinPerTick values

Jok1o7 committed:
Added values for track width

Jok1o7 committed:
Merge branch 'Roadrunner' of https://github.com/edinagoinglikehotcakes/centerstage into Roadrunner

Jok1o7 committed:
Added image processors

rhsftc committed:
Updating rhs

Jok1o7 committed:
Update AutoTestEOCV.java

AlexanderBecker13 committed:
Merge branch 'Roadrunner' into development

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

rhsftc committed:
Merge branch 'development' into rhs

rhsftc committed:
Recent update from RR 1.0

rhsftc committed:
Moved to the HotCakes package.

rhsftc committed:
Put camera in auto.

Jok1o7 committed:
IMU changes and Auto test changes

Jok1o7 committed:
changed RR values

Jok1o7 committed:
DcMotor ->> DcMotorEx

Jok1o7 committed:
added arm limits

Jok1o7 committed:
Creates the Turn arm and set constraints for the rotational arm motor

Jok1o7 committed:
Gamepad EX from FTC lib added and now the rotational motor works

rhsftc committed:
Merge remote-tracking branch 'origin/development' into rhs

# Conflicts:
#	TeamCode/src/main/java/hotcakes/CenterstageDrive.java
#	TeamCode/src/main/java/hotcakes/MotorControl.java
#	TeamCode/src/main/java/hotcakes/RobotHardware.java
#	TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

rhsftc committed:
Building autonomous paths using camera

rhsftc committed:
Set arm motor to use built-in PIDF

rhsftc committed:
Added menu options, filled in some more RR paths.

rhsftc committed:
Fixed the meepmeep paths.

Jok1o7 committed:
ArmMotor --> TurnMotor

Jok1o7 committed:
removed a problem by identifying imu in robot hardware

AlexanderBecker13 committed:
Fixed menu options

AlexanderBecker13 committed:
Merge branch 'rhs' of https://github.com/edinagoinglikehotcakes/centerstage into rhs

rhsftc committed:
Changes to allow sending telemetry from the op mode while menus are running.

Jok1o7 committed:
Updated and put in ENUM for autoState

AlexanderBecker13 committed:
Coded Spike Pixel Placements

rhsftc committed:
Merge branch 'development' into rhs

rhsftc committed:
Made saturation levels public. Moved rectangles.

rhsftc committed:
Wait for camera to open. Display saturation levels.

rhsftc committed:
Added frame capture.

rhsftc committed:
Big re-factor for creating actions.

AlexanderBecker13 committed:
Added code for Red Right and Blue Left

Jok1o7 committed:
Code works

Jok1o7 committed:
Update BasicAuto.java

rhsftc committed:
Update RR for new robot.

Jok1o7 committed:
put the arm at BRAKE when in STOP

rhsftc reviewed (commented):
This might help, but RUN_TO_POSITION mode would  actively try to keep it at 0. Mechanical lock is the best guarantee.

Jok1o7 committed:
Programed the arm to go up and down in the linear way

Haven't tested this code yet so it might not work or it might need tweaking.

rhsftc committed:
Merge remote-tracking branch 'origin/development' into rhs

rhsftc committed:
Put camera into auto white balance and wait to start streaming.

rhsftc committed:
Put camera into auto white balance and wait to start streaming.

rhsftc committed:
Changes to only look at left 2 spikes.

AlexanderBecker13 committed:
Still Trying to get Arm Extender Motor to work

Jok1o7 committed:
I think the arm didn't work not because of the power, but because the triggers are programmed wrong. Maybe put a normal Trigger in instead of a FTC lib thing, because we don't know if the trigger code works.

rhsftc committed:
Running arm motor with triggers works.
Stop and reset encoders needs to be followed by setting a run mode.

Jok1o7 committed:
Are these correct? Or just "Servo"? Added the servos for the grippers

Jok1o7 committed:
Added gripper state and limit

Jok1o7 committed:
Coded the gripper. Changed "ServoControllerEx" to just "Servo. Still need to tweak the numbers of the gripper limit.

rhsftc committed:
TeleOp and Gripper Updates

Used gm0 mecanum drive.  Changed Servo to FTCLIB SimpleServo.

rhsftc committed:
Merge branch 'rhs' into development

Jok1o7 committed:
The arm servo that moves the arm up and down coded *still needs testing*

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
CODED THE GRIPPER FLIPPER

rhsftc committed:
Moved boxes, fixed camera bug

rhsftc committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Coded stuff

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
Add ed code for placing pixel on spike

Jok1o7 committed:
Latest code

Jok1o7 committed:
Hopefully this will make the driving good again

Jok1o7 committed:
@Disabled many unused files so it doesn't crowd the driver hub. Does disabling Mecanum drive affect the running of autonomous?

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Gripper values work

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
Update BasicAuto.java

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
changes for prior lake

Jok1o7 committed:
motor directions

Jok1o7 committed:
uncommited changes

Jok1o7 committed:
remove @disableed

rhsftc committed:
Gradle update

Jok1o7 committed:
MeepMeep update

rhsftc committed:
Added servo tuner

AlexanderBecker13 committed:
Coded ArmMotor

Jok1o7 committed:
Some changes

Jok1o7 committed:
Coded LAUNCHINGSERVO

rhsftc committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Changes to make the motors and servos for the arm hang and launching work

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Ctl + Alt + L

Jok1o7 committed:
Disable servo when hanging

Jok1o7 committed:
Changes to values and the arm servo works now and it disables

AlexanderBecker13 committed:
Camera Tuning, misc. cleanup

AlexanderBecker13 committed:
Went back to old auto

Jok1o7 committed:
Auto works

AlexanderBecker13 committed:
Code for park in backstage

Jok1o7 committed:
Auto works

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
cleaned up driving

Jok1o7 committed:
autonomous and gradle changes

AlexanderBecker13 committed:
Roadrunner Updates

rhsftc committed:
Upgrade Gradle plugin to 8.2

rhsftc committed:
Gradle Update Stuff

Jok1o7 committed:
auto drive april tag

Jok1o7 committed:
Updated launch servo positions

Jok1o7 committed:
Gripper commented back in, values still not tuned

Jok1o7 committed:
gripper open and close position not tested yet but hopefully tuned

Jok1o7 committed:
Changed numbers again, still not testted

Jok1o7 committed:
/

AlexanderBecker13 committed:
Merge branch 'development' into Roadrunnerupdate

AlexanderBecker13 committed:
Roadrunner Update

Jok1o7 committed:
Winch motor positions updated not tested yet

AlexanderBecker13 committed:
Merge branch 'development' into Roadrunnerupdate

Jok1o7 committed:
a

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Tested numbers for grippers

AlexanderBecker13 committed:
Cleaned up and refactored Code

AlexanderBecker13 committed:
Fixed an Error

AlexanderBecker13 committed:
Added Gripper angle

Jok1o7 committed:
-

Jok1o7 committed:
new code still need to tune

Jok1o7 committed:
Arm positions tuned for driving and pickup and gripper angle

AlexanderBecker13 committed:
Tuned gripper values

AlexanderBecker13 committed:
Tuned arm angles

Jok1o7 committed:
changed values for gripper stuff

Jok1o7 committed:
tuned arm numbers

Jok1o7 committed:
Cleaned up controls

AlexanderBecker13 committed:
Coded LaunchAngle

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Launch angles

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
wait launch position

Jok1o7 committed:
a

AlexanderBecker13 committed:
Added new april tag class

Jok1o7 committed:
changed values works now

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
April Tag Launch

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
IMU

rhsftc committed:
Clean up hardware map and some comments.

Jok1o7 committed:
Changes to values of the gripper, arm angle, flipper, etc.

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
Tuned the robot

AlexanderBecker13 committed:
-

Jok1o7 committed:
Position changes

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

rhsftc committed:
Added imu orientation op mode to make sure the imu is working.

Jok1o7 committed:
Changed values to gripper pos

Jok1o7 committed:
Changed auto values that work

Jok1o7 committed:
Change start pos and red left still not fully tuned

Jok1o7 committed:
value changes and added initialization of  the robot hardware the i forgot to commit

rhsftc committed:
Clean up code formatting.

rhsftc committed:
More code clean up.

rhsftc committed:
Clean up formatting.

rhsftc committed:
Remove unnecessary todo's.

rhsftc committed:
Code cleam up.

rhsftc committed:
Put auto and tele in "Competition" group for driver station.

AlexanderBecker13 committed:
Blue code

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
Updated Blue code

Jok1o7 committed:
Change in per tick works now

AlexanderBecker13 committed:
Finished Blue right

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
upgraded blue code

Jok1o7 committed:
Changes To blue left

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

rhsftc committed:
Fix misnamed variable.

AlexanderBecker13 committed:
Added arm extension class

Jok1o7 committed:
created gripper class for actions

Jok1o7 committed:
Created gripper angle branch

Jok1o7 committed:
Created arm lift class

AlexanderBecker13 committed:
added classes to init

Jok1o7 committed:
Auto stufff works for red left left and middle

Jok1o7 committed:
Gripper angle changes

Jok1o7 committed:
Added stuff for arm extension, lift, and gripper angle classes. I might've fixed the run to position error we were having

Because under arm angle state it was only setting the arm angle target position and making the arm motor run to position so i changed that hopefully it was the problem.

Jok1o7 committed:
Edited values for red side auto

Jok1o7 committed:
Changed some values of the blue side to match the red side

Jok1o7 committed:
Values that work (red left right not tested but should be right)

Jok1o7 committed:
Small change

Jok1o7 committed:
Coded all of backdrop to respective locations without any april tags used.

Ron I don't know if i did this correctly but look in the backdrop code area i have a  ".stopAndAdd(getackdropActions())" and click on that I don't know if it works or not. I also created a different one and its a public class like the lift class but inside of basic auto. The class is not used because i don't know if it would work but tell me which of the two, the getBackdropActions or the Backdrop class will would work. Otherwise i can just copy and paste the backdrop arm stuff in there without doing the extra stuff.

Jok1o7 committed:
Actually run it

Jok1o7 committed:
Maybe we need this?

AlexanderBecker13 committed:
Started April Tags

AlexanderBecker13 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

Jok1o7 committed:
Auto works for red left and right blue not tested yet

Jok1o7 committed:
Merge branch 'development' of https://github.com/edinagoinglikehotcakes/centerstage into development

AlexanderBecker13 committed:
Made the code look better

AlexanderBecker13 committed:
Added Launch class from RHS to development

AlexanderBecker13 committed:
Updated drone launch in centerstage

Jok1o7 committed:
Changes to blue side

AlexanderBecker13 committed:
tuned auto Blue left works

AlexanderBecker13 committed:
Tuned angles

AlexanderBecker13 committed:
drone launcher
